test/fixtures/poro.rb

```diff
@@ -61,6 +61,14 @@ def cache_key
   end
 end
 
+class Employee < ActiveRecord::Base
+  has_many :pictures, as: :imageable
+end
```

Comment:
Maybe seializer name can describe what it's for testing?  Polymorhipichasmany polymorphic belongsto